ui/apps/platform/src/Components/SearchFilterInput/SearchFilterInput.tsx

```diff
@@ -9,7 +9,7 @@ type SearchFilterInputProps = {
     className: string;
     handleChangeSearchFilter: (searchFilter: SearchFilter) => void;
     placeholder: string;
-    searchCategory: SearchCategory;
+    searchCategory?: SearchCategory;
```

Comment:
The backend API supports requests without a search category, so I've updated this prop to be optional.